work_lights.py

```diff
@@ -70,6 +71,18 @@ def SetGVCColor(lights):
           for light in LIGHTS]
   return
 
+def SetLightMode(philLights, mode):
```

Comment:
For consistency, maybe update all "lights" objects to philLights?

I assume this is to disambiguate from the LIGHTS constant.